homeworks/04-css/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Personal website about Victor Alvarado.">
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:site" content="@victoralvarado" />
+    <meta name="twitter:creator" content="@victoralvarado" />
+    <meta name="og:type" property="og:type" content="website" />
+    <meta name="og:url" property="og:url" content="https://www.victoralvarado.com/">
+    <meta name="og:title" property="og:title" content="Victor Alvarado - Software Engineer">
+    <meta name="og:image" property="og:image" content="https://www.victoralvarado.com/avatar" />
+    <meta name="og:image:type" property="og:image:type" content="image/svg+xml" />
+    <meta name="og:image:width" property="og:image:width" content="216" />
+    <meta name="og:image:height" property="og:image:height" content="216" />
+    <meta name="og:image:alt" property="og:image:alt" content="Young man with the hat" />
+    <meta name="og:description" property="og:description"
+        content="Personal website about Victor Alvarado as a Software Engineer. Portfolio with skills.">
+    <title>Victor Alvarado - Software Engineer</title>
+    <link rel="stylesheet" href="./styles/style.css">
+</head>
+
+<body>
+    <div class="content">
```

Comment:
You added margin auto for centering and max-width to every section inside the ".content" but it is better to add them once for ".content" itself.